decidim-core/app/views/decidim/open_data/index.html.erb

```diff
@@ -41,6 +41,20 @@
       <% end %>
     </div>
 
+    <div class="editor-content">
+      <ul>
+        <% open_data_component_manifests.each do |manifest| %>
+          <li><%= link_to(t("download_resource", scope: "decidim.open_data.index", resource_name: manifest.name), open_data_download_resource_path(manifest.name)) %></li>
+        <% end %>
+      </ul>
+
+      <ul>
```

Comment:
Maybe we could add some kind of header like "Space data" , like it is generated in the Readme.